packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -357,11 +363,10 @@ function getCompassBinPath({ appPath, packagerOptions: { name } }) {
 /**
  * @param {ExtendedApplication} app
  */
-function addDebugger(app) {
+async function addDebugger(app) {
```

Comment:
This doesn't do anything async, right?

```suggestion
function addDebugger(app) {
```